rules-tests/Php85/Rector/FuncCall/OrdSingleByteRector/Fixture/multibyte_int_var.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php 
+namespace Rector\Tests\Php85\Rector\FuncCall\OrdSingleByteRector\Fixture;
+$i = 123;
+echo ord($i);
+?>
+-----
+<?php 
+namespace Rector\Tests\Php85\Rector\FuncCall\OrdSingleByteRector\Fixture;
+$i = 123;
+echo ord($i[0]);
```

Comment:
this is invalid for integer, only works on string, see https://3v4l.org/2U1F6

Reply:
Ok